CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+This repository contains backports of the CPython `typing` module to earlier versions of
+Python. As such, follow CPython's contribution guidelines as much as possible when
```

Comment:
I think ``follow CPython's contribution guidelines`` is slightly vague for newcomers. There probably isn't a need to create a bpo issue, link it, add news etc. 😉 . However, I haven't been contributing long enough to ``typing_extensions`` so I don't know what would be better. The only thing I can see that _is_ required is to sign the CLA.

Reply:
I guess I was mostly thinking of code style. Let me make this more specific.